nvme-ioctl.c

```diff
@@ -178,6 +178,13 @@ int nvme_compare(int fd, __u64 slba, __u16 nblocks, __u16 control, __u32 dsmgmt,
 		       reftag, apptag, appmask, data, metadata);
 }
 
+int nvme_verify(int fd, __u64 slba, __u16 nblocks, __u16 control,
+	      __u32 reftag, __u16 apptag, __u16 appmask)
+{
+	return nvme_io(fd, nvme_cmd_verify, slba, nblocks, control, 0,
```

Comment:
The kernel only allows read, write, and compare with this function's ioctl, right?

Reply:
I'll have to double check. I have two versions of this patch; one that uses the "deprecated" ioctl which requires a kernel change (which I posted but was rejected) and one that uses the passthrough ioctl. I may have posted the wrong patch. Thanks.